commons/src/main/java/no/ntnu/beardblaster/commons/CreateDocumentFailureException.kt

```diff
@@ -0,0 +1,4 @@
+package no.ntnu.beardblaster.commons
+
+class CreateDocumentFailureException(cause: Throwable?) : Exception(cause) {
```

Comment:
Verbet *Create* i klassenavnet skurrer litt for meg. Ser at den brukes som exception når `create()`-funksjonen feiler og navnet er jo beskrivende sånn sett. Jeg tror det funker bedre om *Create* sløyfes, til f.eks. `DocumentFailureException` eller enda kortere:
```suggestion
class DocumentException(cause: Throwable?) : Exception(cause) {
```

Feilmeldingen vil jo uansett beskrivet *hva konkret* det er som feiler.